mlos_bench/mlos_bench/tests/config/schemas/optimizers/test-cases/bad/invalid/bad_space_adapter_type.jsonc

```diff
@@ -2,7 +2,7 @@
     "class": "mlos_bench.optimizers.mlos_core_optimizer.MlosCoreOptimizer",
 
     "config": {
-        "minimize": "score",
+        "optimization_target": "score",
```

Comment:
Can you add a test where "optimization_direction" is neither "min" nor "max" so we make sure it gets rejected?

Reply:
will do!